docs/Installation.md

```diff
@@ -111,8 +109,51 @@ sudo systemctl reload apache2
 ```
 
 ### 4. FreeBSD
+For all commands below, acquire privileges, i.e. become root:
+``su -l``
+
+* Install Apache ([source](https://www.digitalocean.com/community/tutorials/how-to-install-an-apache-mysql-and-php-famp-stack-on-freebsd-10-1)):
+
+``pkg install apache24``
+
+Enter ``y`` at the confirmation prompt.
+
+This installs Apache and its dependencies.
+
+To enable Apache as a service, add ``apache24_enable="YES"`` to the ``/etc/rc.conf`` file. 
+We will use this sysrc command to do just that:
+``sudo sysrc apache24_enable=yes``
```

Comment:
1. Insert empty line before so that the command comes on its own line.
2. Remove "sudo".